lib/index.js

```diff
@@ -3611,6 +3611,78 @@ Stream.prototype.merge = function () {
 exposeMethod('merge');
 
 /**
+ * Takes a Stream of Streams and merges their values and errors into a
+ * single new Stream, limitting the number of unpaused streams that can
+ * running at any one time.
+ *
+ * Note that no guarantee is made with respect to the order in which
+ * values for each stream end up in the merged stream. Values in the
+ * merged stream will, however, respect the order they were emitted from
+ * their respective streams.
+ *
+ * @id mergeLimit
+ * @section Higher-order Streams
+ * @name Stream.mergeLimit()
+ * @api public
+ *
+ * var txt = _(['foo.txt', 'bar.txt']).map(readFile)
+ * var md = _(['baz.md']).map(readFile)
+ * var js = _(['bosh.js']).map(readFile)
+ *
+ * _([txt, md, js]).mergeLimit(2);
+ * // => contents of foo.txt, bar.txt, baz.txt and bosh.js in the order
+ * they were read, but bosh is not read until the others have comleted
```

Comment:
Typo in "comleted". Also, this wording suggests bosh won't be read until both bar and baz have completed. I suggest "but bosh is not read until either foo/bar or baz has finished.